smodels/tools/theoryPredictionsCombiner.py

```diff
@@ -17,6 +17,7 @@
 from smodels.tools.statistics import CLsfromNLL, determineBrentBracket
 import scipy.optimize as optimize
 from smodels.experiment.exceptions import SModelSExperimentError as SModelSError
+from typing import Text
```

Comment:
> ../smodels/tools/theoryPredictionsCombiner.py:437, shouldnt we have "str" instead of "Text"?

I intentionally use `typing` for annotation because it's more interpretable for the user. You can use `str` it's exactly the same thing just a matter of choice.